wlc/test_wlc.py

```diff
@@ -235,6 +235,60 @@ def test_create_component(self):
                 filemask="po/*.po",
             )
 
+    def test_create_component_local_files(self):
+
+        with open("wlc/test_data/mock/project-local-file.pot") as file:
```

Comment:
Better to avoid hardcoding the path (the working directory can be different when executing tests):

```suggestion
        test_file = os.path.join(os.path.dirname(__file__), "test_data", "mock", "project-local-file.pot")
        with open(test_file) as file:
```